pkg/botController/master.go

```diff
@@ -0,0 +1,85 @@
+package botController
+
+import (
+	"fmt"
+	"gobot.io/x/gobot"
+	"gobot.io/x/gobot/api"
+)
+
+// Command to describe command to be executed on bot
+// Fields:
+// 	Command: 	Command to be executed
+//	Direction: 	Direction to move ollie for commands like Roll, SetHeading, etc
+//		   	set 0 if command doesn't support direction parameter
+//	Speed: 		Speed with which ollie should move for commands like SetMotorValues, Roll, etc
+//			set 0 if command doesn't support speed parameter
+// 	Duration:	Time in Milliseconds for which command will be executed
+//			set 0 to run command for forever
+type Command struct {
+	Command   string
+	Direction int16
+	Speed     uint8
+	Duration  uint16
+}
+
+const (
+	defaultDir       = 0
+	defaultSpeed     = uint8(255)
+	defaultRollSpeed = uint8(100)
+	defaultInterval  = 100
+	defaultDur       = 5000
+)
+
+var (
+	cmdDirection int16
+	cmdSpeed     uint8
+	cmdDuration  uint16
+)
+
+func (olcmd Command) String() string {
+	return fmt.Sprintf("{command:%v direction:%v speed:%v duration:%v}", olcmd.Command, olcmd.Direction, olcmd.Speed, olcmd.Duration)
```

Comment:
Change to log instead. Add String method to olcmd so that string representation of object is printed directly.